doc/guides/keycloak_protection.md

```diff
@@ -1,68 +1,118 @@
 ## Prerequisites:
 
-- You have provisioned Keycloak as described in [AeroGear Docs](https://docs.aerogear.org/aerogear/latest/identity-management.html).
+- There is a Keycloak service available.
+- You must first create a client for your application in the Keycloak Administration Console, and then click on the `Installation` tab, select `Keycloak OIDC JSON` for `Format` Option, and then click on `Download`. Save the downloaded `keycloak.json` file to your project.
 
 ## Setting up Keycloak protection in sync framework
 
-Import Voyager Keycloak module
-```javascript
-const { KeycloakSecurityService } = require('../../packages/apollo-voyager-keycloak')
-```
+1. Import Voyager Keycloak module
+    ```javascript
+    const { KeycloakSecurityService } = require('../../packages/apollo-voyager-keycloak')
+    ```
+2. Read the Keycloak config and pass it to initialise the `KeycloakSecurityService`.
+    ```javascript
+    const keycloakConfig = JSON.parse(fs.readFileSync(path.resolve(__dirname, './path/to/keycloak.json')))
+    const keycloakService = new KeycloakSecurityService(keycloakConfig)
+    ```
+3. Use the `keycloakService` instance to protect your app:
+    ```javascript
+    const app = express()
+    keycloakService.applyAuthMiddleware(app)
+    ```
+4. The last piece of work to do is telling Voyager server that the `keycloakService` above will be used as the security service:
+    ```javascript
+    const voyagerConfig = {
+      securityService: keycloakService
+    }
+    const server = ApolloVoyagerServer(apolloConfig, voyagerConfig)
+    ```
```

Comment:
Could we maybe have a new paragraph (single line) here after the given steps that links to the keycloak example server guide to help give the reader the bigger picture? Example:

```markdown
The [Keycloak Example Server Guide](https://github.com/aerogear/apollo-voyager-server/blob/master/doc/guides/examples.md#keycloak-example) has an example server based off the instructions above and shows all of the steps needed to get it running.
```